node-notes-cli/remove.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+
+const JSONdata = require('./data.json');
+const notes = JSONdata.notes;
+const input = process.argv[3];
+
+const remove = () => {
+  delete notes[input];
+  fs.writeFile('data.json', JSON.stringify(JSONdata, null, 1), err => {
+    if (err) throw err;
+  });
+};
+
+remove();
```

Comment:
Did you mean to call your remove and update functions in their own files too?

Reply:
Oops, didn't mean to leave that there.  Revised!